neqo-transport/src/recovery/mod.rs

```diff
@@ -851,6 +851,17 @@ impl LossRecovery {
         // pto_time to increase which might cause PTO for later packet number spaces to not fire.
         if let Some(pn_space) = pto_space {
             qtrace!([self], "PTO {}, probing {:?}", pn_space, allow_probes);
+            // Packets are only declared as lost relative to `largest_acked`. If we hit a PTO while
+            // we don't have a largest_acked yet, also do a congestion control reaction (because
+            // otherwise none would happen).
```

Comment:
Are we sure that we want to slow down?  If we haven't received an ACK, we should be at a low send rate.  Do we really want to slow down further?

One thing that concerns me here is that a fast PTO (if we ever enable that) will hit this condition more often and that might not be good for performance.  The same goes for long RTT connections, which might be OK while we are still retransmitting within the RTT, but once we get an RTT estimate that is long, we'll slow our send rate by a huge amount for the false PTOs we've hit.

Reply:
I think the argument is that the combination of initial cwnd and initial RTT (= PTO) were made with some sort of safety criteria in mind. A PTO is an indication that either the RTT is much longer than the default assumption, or there is quite a bit of loss. In either of these two cases, we probably want to slow down?

Reply:
I guess my main concern is that if we want to be more aggressive about PTO, such as sending another Initial we create a situation where slowing down is not the result of a misunderstanding of the RTT, but a deliberate choice to send more to start with.